site/en/docs/extensions/reference/webRequest/index.md

```diff
@@ -215,15 +217,15 @@ optionally specify filters and extra information when you register event listene
 ### Request IDs
 
 Each request is identified by a request ID. This ID is unique within a browser session and the
-context of an extension. It remains constant during the the life cycle of a request and can be used
+context of an extension. It remains constant during the life cycle of a request and can be used
 to match events for the same request. Note that several HTTP requests are mapped to one web request
 in case of HTTP redirection or HTTP authentication.
 
 ### Registering event listeners
 
 To register an event listener for a web request, you use a variation on the [usual `addListener()`
 function][10]. In addition to specifying a callback function, you have to specify a filter argument
-and you may specify an optional extra info argument.
+, and you may specify an optional extra info argument.
```

Comment:
Please check the staged rendering of this page and make sure this doesn't render with a space between the previous line and the comma.

Reply:
EEK! Let me fix that! Thanks!! =)